Gemfile

```diff
@@ -68,8 +68,10 @@ group :test do
   gem 'rspec-html-matchers'
   gem 'rubocop', require: false
   gem 'selenium-webdriver'
+  gem 'chromedriver-helper'
   gem 'shoulda-matchers', git: 'https://github.com/thoughtbot/shoulda-matchers.git'
   gem 'simplecov', require: false
   gem 'timecop'
+  gem 'pry-rails'
```

Comment:
Can you also please remove this gem from the Gemfile? I know it's super handy and you're free to use `pry` when working on the app. We try to not have any _"unrelated"_ changes to the actual purpose of a PR (in your case all things addresses).

You can totally update the setup to suit your personal workflow while developing the app - just don't commit these changes when opening a PR.